tests/scripts/keadm_compatibility_e2e.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+KUBEEDGE_ROOT=$PWD
+WORKDIR=$(dirname $0)
+E2E_DIR=$(realpath $(dirname $0)/..)
+IMAGE_TAG=$(git describe --tags)
+EDGECORE_VRRSION=${1:-"1.15.1"}
+KUBEEDGE_VERSION=$IMAGE_TAG
+
+source "${KUBEEDGE_ROOT}/hack/lib/install.sh"
+
+function cleanup() {
```

Comment:
The clean function appears to be redundant code; perhaps it might be referenced in a library directory such as lib/common.sh.

Reply:
There are many changes, can resubmit pr for modification Ensure that the current run is successful